molecule/testinfra/staging/mon/iptables-mon-staging.j2

```diff
@@ -17,14 +17,14 @@
 {% endfor -%}
```

Comment:
Same point as above for the prod iptables template - if it's not updated then these shouldn't be either.

Reply:
:+1: